scripts/multiqc

```diff
@@ -333,6 +342,10 @@ plots_flat, plots_interactive, make_pdf, config_file, verbose, quiet, **kwargs):
         pass # No subdirectory variable given
 
 
+    if config.profile:
```

Comment:
Any reason to start this down here, rather than straight after parsing the command line flags?